eks/eks.go

```diff
@@ -518,13 +522,6 @@ func (ts *Tester) createTesters() (err error) {
 			K8SClient: ts.k8sClient,
 			ECRAPI:    ecr.New(ts.awsSession, aws.NewConfig().WithRegion(ts.cfg.GetAddOnFluentdRepositoryBusyboxRegion())),
 		}),
-		metrics_server.New(metrics_server.Config{
```

Comment:
Let's leave the old metrics serve in here. This is a backwards incompatible change, which will break all consumers (AWSWesleyCI). Typically, that way you want to do this is
1. Introduce the new changes
2. Migrate all consumers to the new way of doing things (this is called "deprecation")
3. Delete the old way of doing things.

Reply:
Yeah let's see if we can keep two versions with shared code.